git_repo/repo.py

```diff
@@ -426,17 +426,17 @@ def request_edition(repository, from_branch, onto_target):
 
         service = self.get_service(resolve_targets=('upstream', '{service}', 'origin'))
 
-        new_request = service.request_create(self.namespace,
+        output_generator = service.request_create(self.namespace,
                 self.repo_name,
                 self.local_branch,
                 self.remote_branch,
                 self.title,
                 self.message,
                 self._auto_slug,
                 request_edition)
-        log.info('Successfully created request of `{local}` onto `{project}:{remote}`, with id `{ref}`!'.format(**new_request))
-        if 'url' in new_request:
-            log.info('available at: {url}'.format(**new_request))
+
+        print_iter(output_generator)
```

Comment:
you can remove the useless intermediate variable (it does not add much to readability):

```python
print_iter(service.request_create(
        self.repo_name,
        self.local_branch,
        self.remote_branch,
        self.title,
        self.message,
        self._auto_slug,
        request_edition
    )
)
```

Reply:
Done!